Cargo.toml

```diff
@@ -8,7 +8,7 @@ repository = "https://github.com/tldr-pages/tlrc"
 documentation = "https://tldr.sh/tlrc"
 license = "MIT"
 edition = "2021"
-rust-version = "1.74"
+rust-version = "1.75"
```

Comment:
This was required for [`File::set_modified`](https://doc.rust-lang.org/std/fs/struct.File.html#method.set_modified)